nltk/app/srparser_app.py

```diff
@@ -59,9 +59,7 @@
       [h]\t Help
       [Ctrl-p]\t Print
       [q]\t Quit
-"""
 
-"""
```

Comment:
hmm yeah good point. These comments are over a decade old and I'm not sure they still apply... @stevenbird?